Source/ProceduralFairings/FairingBase.cs

```diff
@@ -94,6 +94,8 @@ public enum BaseMode { Payload, Adapter, Plate, Other }
         [KSPField] public float decouplerMassMult = 1;              // Mass multiplier
         [KSPField] public float decouplerMassBase = 0;              // Flat additional mass (0.001 = 1kg)
 
+        [KSPField] public float maxFairingSize = 0;                 // the "real" maximum width of the fairing, the bulge in the middle
```

Comment:
Needs `isPersistant = true` for the field value to get saved into the craft file / sfs.